content/it/infrastructure_as_code.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Infrastructure as Code (IaC - Infrastruttura come Codice)
+status: Feedback Appreciated 
```

Comment:
If this entity is okay to be exposed, then please change the `status` to `Completed`:

```suggestion
status: Completed
```

Reply:
@jihoon-seo thank you. Go, go, go!